tests/campaigns/test_integration.py

```diff
@@ -34,11 +54,20 @@ def assert_sub_returned(self, response, campaign_id, budget,
         self.assertEqual(response['subscription']['budget'], budget)
 
 
+class BaseSearchCampaignSubscriptionTestCase(BaseSubscriptionTestCase):
+    campaign_type = 'search'
+
+
+class BaseDisplayCampaignSubscriptionTestCase(
+        BaseSubscriptionTestCase):
+    campaign_type = 'display'
```

Comment:
We should probably define `campaign_type` where it's needed - think it will be more concise